site/en/r2/tutorials/keras/basic_classification.ipynb

```diff
@@ -349,6 +349,34 @@
         "train_labels"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "cIAcvQqMCaWf_"
+      },
+      "source": [
+        "Fashion MNIST dataset is an example of a balanced dataset with 6000 images by label (TRAIN)."
```

Comment:
Fashion mnist has 60000 train images and 10000 test image (https://github.com/zalandoresearch/fashion-mnist)

Reply:
Thank you @yashk2810 
Nice catch, effectively fashion mnist has 60,000 train images and 10,000 test image 
> len(train_labels)
60000
> len(test_labels)
10000